PhysicsTools/PatAlgos/python/producersHeavyIons/heavyIonJets_cff.py

```diff
@@ -70,6 +70,15 @@
     patJetCorrFactors
 )
 
+from Configuration.ProcessModifiers.run2_miniAOD_pp_on_AA_103X_cff import run2_miniAOD_pp_on_AA_103X
+from RecoJets.JetProducers.ak4PFJets_cfi import ak4PFJets,ak4PFJetsCHS
+ak4PFJets.src = "pfEmptyCollection"
+ak4PFJetsCHS.src = "pfEmptyCollection"
+_recoPFJetsHIpostAODTask = recoPFJetsHIpostAODTask.copy()
+_recoPFJetsHIpostAODTask.add(ak4PFJets,ak4PFJetsCHS)
```

Comment:
```suggestion
from CommonTools.ParticleFlow.pfNoPileUpJME_cff import *
_recoPFJetsHIpostAODTask = recoPFJetsHIpostAODTask.copy()
_recoPFJetsHIpostAODTask.add(ak4PFJets, pfNoPileUpJMETask, ak4PFJetsCHS)
```

this will add the other modules needed to rerun ak4PFJetsCHS.
It looks like incidentally some other parts of `PAT` pull them in and miniAOD works, while the PF2PAT breaks.